### PR TITLE
Add loss weights feature for jax backend

### PR DESCRIPTION
**Pull request by bonneted**

### CODE REVIEW

File: deepxde/model.py [hunk 1]
```diff
@@ -402,6 +400,8 @@ def outputs_fn(inputs):
             if not isinstance(losses, list):
                 losses = [losses]
             losses = jax.numpy.asarray(losses)
+            if self.loss_weights is not None:
```
Comment (lululxvi) [inline deepxde/model.py:403]: Update the TODO comment in line 399
